uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/toil-leave-request-instance_test.js

```diff
@@ -114,5 +115,19 @@ define([
         expect(instance.toAPI).toHaveBeenCalled();
       });
     });
+
+    describe('updateDuration()', function () {
+      beforeEach(function () {
+        instance = TOILRequestInstance.init({}, false);
+        instance.toilDurationHours = 1;
+        instance.toilDurationMinutes = 1;
+        promise = instance.updateDuration();
```

Comment:
Still you have `promise = instance.updateDuration();`,
it should be just `instance.updateDuration();`